### ISSUE TITLE
[Failure] value setApplicationProtocols is not a member of javax.net.ssl.SSLParameters

### ISSUE THREAD

**Issue by ignasi35** [akka/akka-http#3251]
seen in https://jenkins.akka.io:8498/job/akka-http-nightly/160/AKKA_VERSION=master,Node=akka-http-nightly-node,SCALA_VERSION=2.13.1,jdk=AdoptOpenJDK%2011/console (can reproduce locally).

```
[info] Compiling 138 Scala sources and 8 Java sources to /home/jenkins/workspace/workspace/akka-http-nightly/AKKA_VERSION/master/Node/akka-http-nightly-node/SCALA_VERSION/2.13.1/jdk/AdoptOpenJDK 11/akka-http/target/scala-2.13/classes ...
[error] /home/jenkins/workspace/workspace/akka-http-nightly/AKKA_VERSION/master/Node/akka-http-nightly-node/SCALA_VERSION/2.13.1/jdk/AdoptOpenJDK 11/akka-http2-support/src/main/scala/akka/http/scaladsl/Http2.scala:195:14: value setApplicationProtocols is not a member of javax.net.ssl.SSLParameters
[error]       params.setApplicationProtocols(Array("h2"))
[error]              ^
[error] one error found
```

It's a weird issue since `setApplicationProtocols` was introduced in `javax.net.ssl.SSLParameters` on java 8.

**Comment by ignasi35** [comment]
I'm suspicious of https://github.com/akka/akka-http/blob/529d33b31d8bd64c7dbc382620bde570adf1cd4f/project/Common.scala#L33-L40 but I'm more confused by the fact that `setApplicationProtocols` was available in JDK8.

**Comment by ignasi35** [comment]
>  but I'm more confused by the fact that `setApplicationProtocols` was available in JDK8.

IT was backported in some versions of JDK8 _and_ documented as being available in JDK8, but JDK11 running with `-release 8` doesn't _see_ it.

**Comment by ignasi35** [comment]
This failure snowballed into other jobs: https://jenkins.akka.io:8498/job/akka-http-docs/445/

**Comment by jrudolph** [comment]
Fixed in #3256
